### DISCUSSION TITLE
Failed to open device

### DISCUSSION THREAD

**Discussion by Julgeh1234** [jvde-github/AIS-catcher#291] [Q&A]
I already installed AIScatcher on my Raspberry Pi 4 Model B. I plugged in the RTL-SDR Blog V4 and everything was fine. But now I got into trouble when i tied to execute this in my console: `AIS-catcher -v 10 -X -u 127.0.0.1 10110 -u 127.0.0.1 10111` 

I got the error:

 `usb_claim_interface error -6
Error: RTLSDR: cannot open device.
UDP: close socket for host: 127.0.0.1, port: 10110
UDP: close socket for host: 127.0.0.1, port: 10111
`

Can someone help me please? I dont know how to move on... THX

I also tried: `rtl_test -t`
->the exact same error.

**Comment by jvde-github** [comment]
It is probably already running in the background? 

How did you install it?

It probably can be disabled but good to understand how you installed. FWIW, If it is with abcd567 script, use: sudo systemctl stop aiscatcher.service

**Comment by Julgeh1234** [comment]
I got the solution. In the background there was still a rtl_test task running. That blocked and AIScatcher had no permission.

**Reply by jvde-github** [reply]
Great, glad it is resolved.
